sdk/resourcemanager/azure-resourcemanager-privatedns/src/main/java/com/azure/resourcemanager/privatedns/implementation/PrivateDnsZoneImpl.java

```diff
@@ -454,4 +473,17 @@ public Update withETagCheck(String etagValue) {
                 }
             });
     }
+
+    private String serializeTags(Map<String, String> tags) {
+        try {
```

Comment:
Can you just do a new HashMap? It does not really need this kind of JSON, at least not now.

Reply:
Do you mean create a deep copy or have add/remove maps?

Reply:
There is no deep copy for a HashMap, you just create a new one via constructor.

Its content is String, which is immutable.